src/events/WebhookEvent.ts

```
@@ -2,12 +2,16 @@ import type { Logger } from '@credo-ts/core'
 
 import fetch from 'node-fetch'
 
+const DEFAULT_TIMEOUT = 5000;
```

ESLint

[error] Delete `;`

(prettier/prettier)